notebooks/multi_modal/solutions/image_captioning.ipynb

```diff
@@ -0,0 +1,1136 @@
+{
```

Comment:
<p>Can you add a note on approximately how long it will take for 1 epoch on the t4 GPU? Just so the the students know what to expect! </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/asl-ml-immersion/pull/226/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='41'/>

Reply:
<p>Makes sense! I added it.</p>